### DISCUSSION TITLE
Fork again

### DISCUSSION THREAD

**Discussion by kaiquekandykoga** [community/community#28420] [Other Feature Feedback, Questions, & Ideas]
Let's assume the following scenario.

[Owner 00] Repository A.

[Owner 01] Repository B is a fork of [Owner 00] Repository A.

If I have a fork of [Owner 01] Repository B, I will have [Owner myself] Repository C.

Sometimes there are repositories that are forking other repositories to create new projects from the forked projects, and not to send patches in a forking workflow to the original repository. If I am trying to contribute to Repository A and Repository B, I cannot fork both repositories by clicking the fork button at the same time. In the above example I am not allowed to fork Repository A, because GitHub is assuming that I already have a fork of it. It would be nice to be able to fork multiple times per user.

Workaround 00 > Create a new organization, and fork the repository for that organization. Not the best experience I think.

Workaround 01 > Delete my fork from Repository B, and after that fork Repository A. Again not a nice experience.

Thank you.

**Comment by jfullstackdev** [comment]
simply create an Org, plus it has the dedicated Dashboard for all of its activities, even if you don't have members, because as you can see from my Org, I can fork my own repo in that Org and I can fork the fork from the same repo

![image](https://user-images.githubusercontent.com/47092464/183002254-8648d662-faf5-430e-b5e4-326c20ec1969.png)

this is intended to address users within that Org but you can use that for your own benefit, enjoy !

**Reply by kaiquekandykoga** [reply]
I appreciate your help. Thank you for taking your time answering.

I agree that it is a solution, still I think that if I could fork another fork and so on without creating an organization would be better.
